docs/install-upgrade/build-wiring.md

```diff
@@ -3,36 +3,74 @@
-   hhfab wiring sample - sample wiring diagram (would work for vlab)
+   hhfab sample - generate sample wiring diagram
 
 USAGE:
-   hhfab wiring sample [command options] [arguments...]
+   hhfab sample command [command options]
+
+COMMANDS:
+   spine-leaf, sl      generate sample spine-leaf wiring diagram
+   collapsed-core, cc  generate sample collapsed-core wiring diagram
+   help, h             Shows a list of commands or help for one command
 
 OPTIONS:
-   --brief, -b                    brief output (only warn and error) (default: false)
-   --fabric-mode value, -m value  fabric mode (one of: collapsed-core, spine-leaf) (default: "spine-leaf")
-   --help, -h                     show help
-   --verbose, -v                  verbose output (includes debug) (default: false)
-
-   wiring generator options:
-
-   --chain-control-link         chain control links instead of all switches directly connected to control node if fabric mode is spine-leaf (default: false)
-   --control-links-count value  number of control links if chain-control-link is enabled (default: 0)
-   --fabric-links-count value   number of fabric links if fabric mode is spine-leaf (default: 0)
-   --mclag-leafs-count value    number of mclag leafs (should be even) (default: 0)
-   --mclag-peer-links value     number of mclag peer links for each mclag leaf (default: 0)
-   --mclag-session-links value  number of mclag session links for each mclag leaf (default: 0)
-   --orphan-leafs-count value   number of orphan leafs (default: 0)
-   --spines-count value         number of spines if fabric mode is spine-leaf (default: 0)
-   --vpc-loopbacks value        number of vpc loopbacks for each switch (default: 0)
+   --help, -h  show help
 ```
+
+## Design Discussion
+This section is meant to help the reader understand how to assemble the primitives presented by the Fabric API into a functional fabric. This discussion starts with the primary building block of your design, a VPC.
+
+### VPC
+
+A VPC allows for isolation at layer 3. This is the main building block for users when creating their architecture. Hosts inside of a VPC will see each other but nothing else. The hosts inside of a VPC will likely need to connect to other VPCs or the outside world. To communicate between two VPC a *peering* will need to be created. A VPC can be a logical separation of workloads. By separating these workloads additional controls are available. The logical separation doesn't have to be the traditional database, web, and compute layers it could be development teams who need isolation, it could tenants inside of an office building, or any separation that allows for better control of the network. Once your VPCs are decided, the rest of the fabric will come together. With the VPCs decided traffic can be prioritized, security can be put into place, and the wiring can begin. The fabric allows for the VPC to exist beyond a single switch, which gives flexibility when the physical world meets the digital.
```

Comment:
```suggestion
A VPC allows for isolation at layer 3. This is the main building block for users when creating their architecture. Hosts inside of a VPC will see each other but nothing else. The hosts inside of a VPC will likely need to connect to other VPCs or to the outside world. To communicate between two VPCs, a *peering* will need to be created. A VPC can be a logical separation of workloads. By separating these workloads, additional controls are available. The logical separation doesn't have to be the traditional database, web, and compute layers; it could be development teams who need isolation, or tenants inside of an office building, or any separation that allows for better control of the network. Once you have decided how to setup your VPCs, the rest of the fabric will come together. You can then prioritize traffic, put security policies into place, and begin the wiring. The fabric allows for the VPC to exist beyond a single switch, which gives flexibility when associating physical hardware to your cloud infrastructure.
```

I'd avoid passive tense and figures of speech as much as possible. I'm not very happy with my suggestion for the end of the paragraph though, there's certainly something better to do.

Note: I've got a personal preference for wrapping the text. Most of the sources are wrapped, as far as I can remember.

Reply:
Hosts inside of a VPC will see each other but nothing else.
should be 
host insides of  vpc belong to the same broadcast domain and can see each other.